Makefile

```diff
@@ -293,57 +291,47 @@ release-notes: $(RELEASE_NOTES)
-	kubectl \
-		wait --for=condition=Ready --timeout=5m -n capi-system pod -l control-plane=controller-manager
-    # Wait for capz-controller
-	kubectl \
-		wait --for=condition=Ready --timeout=5m -n capz-system pod -l control-plane=capz-controller-manager
+	# Deploy CAPI
+	kubectl apply -f https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.3.0-rc.3/cluster-api-components.yaml
+
+	# Deploy CAPZ
+	kustomize build config | $(ENVSUBST) | kubectl apply -f -
+
+	# Wait for CAPI pods
+	kubectl wait --for=condition=Ready --timeout=5m -n capi-system pod -l cluster.x-k8s.io/provider=cluster-api
+	kubectl wait --for=condition=Ready --timeout=5m -n capi-kubeadm-bootstrap-system pod -l cluster.x-k8s.io/provider=bootstrap-kubeadm
+	kubectl wait --for=condition=Ready --timeout=5m -n capi-kubeadm-control-plane-system pod -l cluster.x-k8s.io/provider=control-plane-kubeadm
+
+	# Wait for CAPZ pods
+	kubectl wait --for=condition=Ready --timeout=5m -n capz-system pod -l cluster.x-k8s.io/provider=infrastructure-azure
+	
 	# Create Cluster.
 	sleep 10
+	kustomize build templates | $(ENVSUBST) | kubectl apply -f -
 
-	# Create Cluster.
-	kubectl \
-		create -f examples/_out/cluster.yaml
-	# Create control plane machines.
-	kubectl \
-		create -f examples/_out/controlplane.yaml
-	# wait for the first control plane machine to be ready
-	./examples/wait-for-ready.sh
-	# Fetch the Kubeconfig for the target cluster
-	source ./examples/_out/.env; \
-	kubectl \
-		--namespace=default get secret/$$CLUSTER_NAME-kubeconfig -o json \
-		| jq -r .data.value \
-		| base64 --decode > ./examples/_out/clusterapi.kubeconfig
-
-	# Deploy a CNI soution, Calico
-	kubectl --kubeconfig=./examples/_out/clusterapi.kubeconfig \
-	  apply -f https://docs.projectcalico.org/v3.8/manifests/calico.yaml
-	# Create 2 worker nodes with MachineDeployment.
-	kubectl \
-		create -f examples/_out/machinedeployment.yaml
+	# Wait for the kubeconfig to become available.
+	timeout 300 bash -c "while ! kubectl get secrets | grep $(CLUSTER_NAME)-kubeconfig; do sleep 1; done"
+	# Get kubeconfig and store it locally.
+	kubectl get secrets $(CLUSTER_NAME)-kubeconfig -o json | jq -r .data.value | base64 --decode > ./kubeconfig
+	timeout 300 bash -c "while ! kubectl --kubeconfig=./kubeconfig get nodes | grep master; do sleep 1; done"
+
+	# Deploy calico
+	kubectl --kubeconfig=./kubeconfig apply -f https://docs.projectcalico.org/v3.13/manifests/calico.yaml
 
 	@echo 'Set kubectl context to the kind management cluster by running "kubectl config set-context kind-clusterapi"'
-	@echo 'run "kubectl --kubeconfig=./examples/_out/clusterapi.kubeconfig ..." to work with the new target cluster'
+	@echo 'run "kubectl --kubeconfig=./kubeconfig ..." to work with the new target cluster'
 
 .PHONY: delete-cluster
 delete-cluster: $(CLUSTERCTL) ## Deletes the example Kubernetes Cluster "clusterapi"
 	# Fetch the Kubeconfig for the target cluster
-	source ./examples/_out/.env; \
+	source ./templates/_out/.env; \
```

Comment:
Will this file exist? I'm not seeing it being created anywhere, but I could very well be missing it.

Reply:
whoops you're right, forgot to change this